File: uecho/ruby/uecho.rb
require 'webrick'
server = WEBrick::HTTPServer.new(:Port => 8080)
server.mount_proc('/') {|request, response|
  path = request.request_uri.path.to_s
  estring = path[(path.start_with?('/') ? 1 : 0)..]
  response.body = "<html><head><title>Unstoppable Echo</title></head><body><h1>#{estring} right back at ya!</h1></body></html>"
}
trap("INT") {server.shutdown}
server.start

File: uecho/golang/uecho.go
// Based on code from https://golang.org/doc/articles/wiki/#tmp_3

package main

import (
    "fmt"
    "log"
    "net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
    fmt.Fprintf(w, "<html><head><title>Unstoppable Echo</title></head><body><h1>%s right back at ya!</h1></body></html>", r.URL.Path[1:])
}

func main() {
    http.HandleFunc("/", handler)
    log.Fatal(http.ListenAndServe(":8080", nil))
}
